src/client/cli/cmd/launch.cpp

```diff
@@ -461,19 +501,9 @@ mp::ReturnCode cmd::Launch::request_launch(const ArgParser* parser)
     return dispatch(&RpcMethod::launch, request, on_success, on_failure, streaming_callback);
 }
 
-auto cmd::Launch::mount_home(const mp::ArgParser* parser) -> ReturnCode
+auto cmd::Launch::mount(const mp::ArgParser* parser, QString mount_source, QString mount_target) -> ReturnCode
```

Comment:
The reason I didn't use `const` references is because, as far as I understand, `QString` uses a copy-on-write implementation, so there shouldn't be any expensive copying.

Reply:
Well, I learned something new :wink:  I didn't realize `QString` was handled this way internally.  That said, I think in order to be consistent with the rest of the code and also to avoid the possibility of introducing argument passing for types that *don't* behave this way, it should still be a `const` reference.  `const` reference certainly won't hurt and maintains const correctness.